apps/opik-frontend/src/api/playground/createLogPlaygroundProcessor.ts

```diff
@@ -112,7 +118,16 @@ const getTraceFromRun = (run: LogQueueParams): LogTrace => {
   return trace;
 };
 
+const hasChoicesContent = (run: LogQueueParams): boolean => {
+  return run?.choices?.some((choice) => choice.delta.content) ?? false;
```

Comment:
🤏 Nit pick:

`!!run?.choices?.some((choice) => choice.delta.content)` would be better